### DISCUSSION TITLE
About startup time of built apps

### DISCUSSION THREAD

**Discussion by liu-wh** [wailsapp/wails#1815] [Q&A] [answered]
Hello, I have observed inconsistent startup times for built apps,  sometimes fast (within 1s) and sometimes very slow(6-7s). 
Can anyone give me some advice? Thanks!

**Comment by liu-wh** [comment, marked as answer]
I found the issue,  because it was blocked by antivirus software.

**Comment by brys0** [comment]
I have a similar issue, it only shows up on the PC i built the app on, otherwise it's quite fast. Startup takes around 200-500ms (10ms preinit, disk write, etc) On another machine startup only takes about 30-100ms. Way faster, not sure why :/
